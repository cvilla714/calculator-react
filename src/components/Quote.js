import React from 'react';

const Quote = () => (
  <div>
    <h1>hello</h1>
  </div>
);

export default Quote;
