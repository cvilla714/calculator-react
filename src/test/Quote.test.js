import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('should display the correct quote on the Quote Page', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
