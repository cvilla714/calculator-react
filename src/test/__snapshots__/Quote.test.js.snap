// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the correct quote on the Quote Page 1`] = `
<div
  className="container-fluid large "
>
  <div
    className="container"
  >
    <h1
      className="mx-5 d-flex justify-content-center centro align-items-center"
    >
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
    </h1>
  </div>
</div>
`;
